### DISCUSSION TITLE
HRRR '6000 ft AGL'?

### DISCUSSION THREAD

**Discussion by adair-kovac** [NOAA-GSL/GSL-Regional-Model-Forum#18] [Q&A] [answered]
The HRRR smoke viewer page 

[https://rapidrefresh.noaa.gov/hrrr/HRRRsmoke/](https://rapidrefresh.noaa.gov/hrrr/HRRRsmoke/)

has a row (currently doesn't seem to be active) for '6000 ft AGL smoke' and '1000 ft AGL smoke'. HRRR levels aren't measured in feet, so this has been confusing. Where does this data come from?

**Comment by EricJames-NOAA** [comment, marked as answer]
Thanks for the question.  We're in the process of transitioning our model graphics from NCAR Command Language (NCL) to Python.  The 1000 ft and 6000 ft AGL smoke graphics are not yet working in the python framework.  

But these fields can be calculated based on the native HRRR model levels...we simply do vertical interpolation of the smoke field to a constant height above ground.

**Reply by adair-kovac** [reply]
Cool, congrats on the upgrades! Do y'all have anything on the roadmap to make it more transparent where the data in the various tools/graphics comes from?

**Reply by EricJames-NOAA** [reply]
We have a description of the various diagnostics in this recent tech memo:

https://rapidrefresh.noaa.gov/Diag-vars-NOAA-TechMemo.pdf

For the smoke component, we have this User's guide which goes into some more detail:

https://rapidrefresh.noaa.gov/hrrr/HRRRsmoke/RAP_HRRR-Smoke_user_guide_Nov2021.pdf
